src/api/config.go

```diff
@@ -42,6 +42,13 @@ type Config struct {
 
 	// Host is a main app host to provide a quick preview for the deployed apps
 	Host string `envconfig:"HOST" required:"true"`
+
+	RegistryTLSVerify bool   `env:"REGISTRY_TLS_VERIFY" default:"false"`
+	RegistryCertDir   string `env:"REGISTRY_CERT_DIR" default:"./certs"`
+	RegistryAuthType  string `env:"REGISTRY_AUTH_TYPE" default:"token"`
```

Comment:
In a docker package you defined 3 possible values for docker auth type.

First, it can be empty string (no auth), so let's add `required:"false"`.
Second, in order to validate the config on start up we can implement custom Decode for this field.

Below is an example for StringBase64. We don't expect any different format (like base64), just to check it's either empty, or one of the 2 [token, basic], if not - return ErrUnknownDockerAuthType (this error is not defined yet)